include/schrodinger/sketcher/public_constants.h

```diff
@@ -22,5 +22,19 @@ enum class SelectionTool {
     FRAGMENT,
 };
 
+/**
+ * Whether the Sketcher can be used to draw atomistic models (i.e. small
+ * molecules), monomeric models (i.e. HELM models), or both
+ */
+typedef int InterfaceTypeType;
+namespace InterfaceType
+{
+enum InterfaceTypeType {
```

Comment:
I think this is fine to leave for now, but I think I have in my head not actually needing this toggle -- the sketcher would always have the option to flip. I guess my question is, why do we ever want MONOMERIC only?

We might want different UX (ketcher does a decent job of having the toggle always there but it not being invasive), but that's a conversation I should have with Laura. This seems fine for now though, esp. since we want to hide monomer stuff for a while